Configuration/TypoScript/constants.txt

```diff
@@ -0,0 +1,108 @@
+############################
+### CUSTOM SUBCATEGORIES ###
+############################
+
+# Predefined categories
+# basic		Constants of superior importance for the template. This is typically dimensions, image files and enabling of various features. The most basic constants, which you would almost always want to configure.
+# menu		Menu setup. This includes font files, sizes, background images. Depending on the menu type.
+# content	All constants related to the display of page content elements.
+# page		General configuration like meta tags, link targets.
+# advanced	Advanced functions, which are seldom used.
+#
+# customcategory=ai100basic=Aimeos web shop: basic
+# customcategory=ai110menu=Aimeos web shop: menu
+# customcategory=ai120content=Aimeos web shop: content
+# customcategory=ai130page=Aimeos web shop: page
+# customcategory=ai140advanced=Aimeos web shop: advanced
+# customcategory=ai150theme=Aimeos web shop: theme
+
+# Predefined subcategories
+# enable	Used for options that enable or disable primary functions of a template.
+# dims		Dimensions of all kinds; pixels, widths, heights of images, frames, cells and so on.
+# file		Files like background images, fonts and so on. Other options related to the file may also enter.
+# typo		Typography and related constants.
+# color		Color setup. Many colors will be found with related options in other categories though.
+# links		Links: Targets typically.
+# language	Language specific options.
+#
+# customsubcategory=ais100test=test
+
+######################
+### AIMEOS PACKAGE ###
+######################
+plugin.tx_aimeos {
+	# cat=ai100basic/enable/100; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-list.catid-default
+	catalog.lists.catid-default =
+	# cat=ai100basic/enable/110; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-detail.prodid-default
+	catalog.detail.prodid-default =
+	# cat=ai100basic/enable/120; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-filter.tree-startid
+	catalog.filter.tree.startid =
+	# cat=ai100basic/enable/121; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-filter.tree-levels-always
+	catalog.filter.tree.levels-always =
+	# cat=ai100basic/enable/122; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-filter.tree-levels-only
+	catalog.filter.tree.levels-only =
```

Comment:
That's right. But I had the case where I need the catid-default has to be set. Else no texts are shown on the list page if no category is selected by param. So I think in minimum this should remain...